resources-service/src/resource/resource-routes-v0.ts

```diff
@@ -42,6 +48,21 @@ const resourceRoutes = () => {
     res.json(referrableResources);
   });
 
+  router.post(
```

Comment:
Keeping a version of this around just until the cloudsearch version is solid, we can switch over to it testing the front end if needed that way